src/main/app/claims/claimModelConverter.ts

```diff
@@ -43,58 +63,47 @@ export class ClaimModelConverter {
 
     draftClaim['feeAccountNumber'] = draftClaim.feeAccount.reference
     delete draftClaim.feeAccount
+    delete draftClaim.representative
```

Comment:
why we need to delete `representative` here?

Reply:
representative is moved to claimant

Reply:
ok.